packages/components/src/List/__snapshots__/List.test.js.snap

```diff
@@ -537,535 +537,508 @@ exports[`List should render 1`] = `
         }
       }
       sort={false}
+      t={[Function]}
     >
-      <Toolbar
-        components={Object {}}
-        display={
-          Object {
-            "mode": "table",
-            "onChange": [MockFunction],
-          }
-        }
-        filter={
-          Object {
-            "onFilter": [MockFunction],
-            "onToggle": [MockFunction],
-          }
-        }
-        i18n={Object {}}
-        pagination={
-          Object {
-            "itemsPerPage": 5,
-            "onChange": [MockFunction],
-            "startIndex": 6,
-            "totalResults": 13,
-          }
-        }
-        sort={false}
-        t={[Function]}
-        tReady={true}
+      <div
```

Comment:
As in a previous PR, this is a simple change that is displayed in an unreadable manner :D 
The i18n HOC mock returns the original component with a modified name `withI18nextTranslation(name)`

Before
```
<withI18nextTranslation(Toolbar)>
    <Toolbar>
        ....
    </Toolbar>
</withI18nextTranslation(Toolbar)>
```

After 
```
<withI18nextTranslation(Toolbar)>
        ....
</withI18nextTranslation(Toolbar)>
```